diff --git a/dist/index.d.ts b/dist/index.d.ts
index 62c974eff5633b6f4ac3fb6e7cb0a95208df291f..76194626ac114ea1acc40a54ff6d6ceb5bca3f8f 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -3,3 +3,6 @@ export { PolicyError, VerificationError } from './error';
 export { KeyFinderFunc, TrustMaterial, toTrustMaterial } from './trust';
 export { Verifier, VerifierOptions } from './verifier';
 export type { SignedEntity, Signer, VerificationPolicy } from './shared.types';
+export { verifyCertificateChain } from './key/certificate';
+export { verifyTLogInclusion } from './tlog';
+export type { CertAuthority, TLogAuthority } from './trust';
diff --git a/dist/index.js b/dist/index.js
index 3222876fcd68b7d7edc5f18e3112c4ae31867f20..80cc8c59e2c052a90756dabea44e826918bd8cfe 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -26,3 +26,7 @@ var trust_1 = require("./trust");
 Object.defineProperty(exports, "toTrustMaterial", { enumerable: true, get: function () { return trust_1.toTrustMaterial; } });
 var verifier_1 = require("./verifier");
 Object.defineProperty(exports, "Verifier", { enumerable: true, get: function () { return verifier_1.Verifier; } });
+var key_1 = require("./key/certificate");
+Object.defineProperty(exports, "verifyCertificateChain", { enumerable: true, get: function () { return key_1.verifyCertificateChain; } });
+var tlog_1 = require("./tlog");
+Object.defineProperty(exports, "verifyTLogInclusion", { enumerable: true, get: function () { return tlog_1.verifyTLogInclusion; } });
